lib/ts/recipe/session/sessionFunctions.ts

```diff
@@ -324,7 +348,7 @@ export async function refreshSession(
     } = {
         refreshToken,
         antiCsrfToken,
-        enableAntiCsrf: handShakeInfo.antiCsrf === "VIA_TOKEN",
+        enableAntiCsrf: !disableAntiCSRF && handShakeInfo.antiCsrf === "VIA_TOKEN",
     };
 
     if (handShakeInfo.antiCsrf === "VIA_CUSTOM_HEADER") {
```

Comment:
The UNAUTHORISED error type has a `clearCookies` flag. This should be renamed to `clearTokens`